include/net/tftp.h

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Please change the commit subject to `net: tftp: ...` in all of your commits.

Reply:
> Hmm, I did not realized that you were creating a new PR so commented the old one (you could just have pushed your fixed tree on top of your older PR). Can you check my comments from the closed PR?

----

Commenting this although you closed it. I would prefer that we have the IP address in the API in protocol agnostic way, so struct sockaddr *server here. Then it is easier to add IPv6 support too.

---
thanks!
yes, sorry something went wrong yesterday... we will fix it.

Reply:
Done..